src/error.rs

```diff
@@ -76,6 +76,11 @@ pub enum Error {
     ScalarMalformed,
 }
 
+/// Result Types with PLONK Errors
+// NOTE: This type is not very useful in this crate until this PR is merged:
+// <https://github.com/arkworks-rs/algebra/pull/350>.
+pub type Result<T = ()> = core::result::Result<T, Error>;
+
```

Comment:
Shouldn't we then leave an issue waiting for the PR instead of adding code that we don't yet need directly?

Reply:
Sorry, that was an oversight. I meant to delete that before pushing.

Reply:
Can you then delete it please? Or you left it due to any other reason? :)

Reply:
I was just waiting for your comments to do a commit.